doc/classes/ProjectSettings.xml

```diff
@@ -810,7 +810,9 @@
 			Main window can't be focused. No-focus window will ignore all input, except mouse clicks.
 		</member>
 		<member name="display/window/size/resizable" type="bool" setter="" getter="" default="true">
-			Allows the window to be resizable by default.
+			If [code]true[/code], allows the window to be resizable by default.
+			[b]Note:[/b] This property is only read when the project starts. To change whether the window is resizable at runtime, set [member Window.unresizable] instead on the root Window, which can be retrieved using [code]get_viewport().get_window()[/code]. [member Window.unresizable] takes the opposite value of this setting.
```

Comment:
This is true for pretty much any project setting (at least window releated).

Reply:
`get_window()` is available in base `Node`, like `get_viewport()`

Reply:
So what should we do here? It seems a bit weird indeed to flag it specifically here when it applies to all other similar settings.

Reply:
> So what should we do here? It seems a bit weird indeed to flag it specifically here when it applies to all other similar settings.

The policy until now has been to add this note to every project setting where relevant (as some *do* have an affect when change at runtime). I agree it gets repetitive, but until we get a macro system for documentation, I'm not sure how we can avoid this while ensuring the documentation is as informative as possible.

Reply:
Hm right, then doing it manually might still be the best option.